### DISCUSSION TITLE
table Extensions

### DISCUSSION THREAD

**Discussion by lzxlzxlzxl** [ueberdosis/tiptap#3123] [General Questions]
Does anyone know how to set the table border color？

**Comment by steFanBzh** [comment]
for example my css:
table {
      border-collapse: collapse;
      table-layout: fixed;
      width: 100%;
      // min-width: 0px!important;
      margin: 0;
      overflow: hidden;

      td {
        border: 2px solid #ced4da;
      }
      th {
        min-width: 1em;
        border: 2px solid #ced4da;
        padding: 3px 5px;
        vertical-align: top;
        box-sizing: border-box;
        position: relative;

        > * {
          margin-bottom: 0;
        }
      }

      th {
        font-weight: bold;
        text-align: left;
        background-color: #f1f3f5;
      }
      .tableWrapper {
        padding: 1rem 0;
        overflow-x: auto;
      }
    }
  }

**Reply by lzxlzxlzxl** [reply]
Thanks for your answer, but I want the method to set the border dynamically instead of writing it in CSS。Do you know how to do that？
